### PR TITLE
Fix updating the log when a TE is created

### LINKED ISSUE #1937: The log doesn't update when creating a TE
Easiest steps to reproduce:
- sign up using the app
- switch to manual mode in settings
- go offline
- create a TE in manual mode
- you won't see your new TE (those fake placeholders will still be there), although the duration of the `Today` section will already include the duration of the first TE

### PR DESCRIPTION
**Pull request by simonrozsival**
Closes #1937 

Squash whenever

### CODE REVIEW

**Comment by kornelijepetak** [comment]
Oh, I see @juanlaube already assigned himself to review :) Good.

File: Toggl.Foundation.MvvmCross/ViewModels/TimeEntriesLogViewModel.cs [hunk 1]
```diff
@@ -92,8 +92,7 @@ public sealed class TimeEntriesLogViewModel : MvxViewModel
             var createDisposable =
                 dataSource.TimeEntries.TimeEntryCreated
                     .Where(isNotRunning)
-                    .Buffer(timeEntryCreationBufferDuration)
-                    .Subscribe(safeInsertTimeEntries);
+                    .Subscribe(safeInsertTimeEntry);
```
Comment (juanlaube) [inline Toggl.Foundation.MvvmCross/ViewModels/TimeEntriesLogViewModel.cs]: `timeEntryCreationBufferDuration` is not referenced anymore.

File: Toggl.Foundation.MvvmCross/ViewModels/TimeEntriesLogViewModel.cs [hunk 2]
```diff
@@ -155,49 +154,34 @@ private void onTimeEntryUpdated((long Id, IDatabaseTimeEntry Entity) tuple)
...
+                safeInsertTimeEntry(timeEntry);
             }
         }
 
-        private void safeInsertTimeEntries(IEnumerable<IDatabaseTimeEntry> timeEntries)
+        private void safeInsertTimeEntry(IDatabaseTimeEntry timeEntry)
         {
-            if (!timeEntries.Any()) return;
-
             IsWelcome = false;
```
Comment (juanlaube) [inline Toggl.Foundation.MvvmCross/ViewModels/TimeEntriesLogViewModel.cs]: Is this some kind of side effect? 😄

**Comment by amulware** [comment]
I reviewed half of these changes earlier. Updating this will make the PR much smaller (though don't forget juan's comment, which we missed on the other one) :)

File: Toggl.Foundation.MvvmCross/ViewModels/TimeEntriesLogViewModel.cs (hunk 2 above)
Comment (simonrozsival) [reply Toggl.Foundation.MvvmCross/ViewModels/TimeEntriesLogViewModel.cs]: Yep (see the `OnIsWelcomeChanged` method). This VM would benefit from some kind of refactoring :)

**Comment by simonrozsival** [comment]
@juanlaube @amulware addressed.

**Comment by amulware** [comment]
Yes, I'm also looking forward to a refactor of this. Especially one that can do batch updates and not raise a whole bunch of redundant events and stuff when multiple time entries are downloaded at once. :D

**Pull request merged**